schema.graphql

```diff
@@ -125,6 +125,16 @@ type BulkIssue @entity {
   updatedAt: BigInt! # block.timestamp when entity last updated
 }
 
+type BulkTransfer @entity {
+  id: ID! # merkleRoot.toHexString()
+  merkleRoot: Bytes! # batch merkle root
+  txs: [Bytes!] # Hashes of ApproveSRRs processed
```

Comment:
`txs` doesn't really reflect what is stored here i think. how about srrApproveHashes as this is what is stored here. the event param name is also `srrApproveHash`

Reply:
I agree with you, fixed it.